Makefile

```diff
@@ -47,6 +47,7 @@ IMAGE_NAME?=network-operator
 CONTROLLER_IMAGE=$(REGISTRY)/$(IMAGE_NAME)
 IMAGE_BUILD_OPTS?=
 BUNDLE_IMG?=network-operator-bundle:$(VERSION)
+BUNDLE_OCP_VERSIONS=v4.9-v4.16
```

Comment:
@rollandf  min OCP should be aligned with our support matrix for network operator 24.7.0 which is min 4.14

Reply:
Done